script/eval.py

```diff
@@ -0,0 +1,21 @@
+from fire import Fire
+from metrics import mse_per_sample, psnr_per_sample, ssim_per_sample
+
+def evaluation_script(y_test, trained_model):
```

Comment:
we can't pass in python objects (trained_model) throguh the command line so: it's better to pass in the image directory
and the checkpoint directory.